website/docs/addon_resolve_admin.md

```diff
@@ -15,96 +15,51 @@ import Tabs from '@theme/Tabs';
+Resolve 17 is using Python 3.6.2
+Resolve 18 is using Python 3.9.x
 
-<Tabs
-  groupId="platforms"
-  defaultValue="win"
-  values={[
-    {label: 'Windows', value: 'win'},
-    {label: 'Linux', value: 'linux'},
-    {label: 'Mac', value: 'mac'},
-  ]}>
 
-<TabItem value="win">
+### Configuring settings for Python home directory
+To be able to run python in Resolve you need to set `RESOLVE_PYTHON3_HOME` environment variable to the directory path of your Python interpreter version.
 
-`%LOCALAPPDATA%\Programs\Python\Python36`
+Open your server Studio settings and go to **Application** addon `ayon+settings://applications/applications/resolve`. Here you can add the path to the **Environment** section.
 
-</TabItem>
-<TabItem value="linux">
+![Ayon/Application/Resolve](assets/resolve_python_home_application_addon.png)
 
-`/opt/Python/3.6/bin`
 
-</TabItem>
-<TabItem value="mac">
+### Installing Python dependencies
 
-`~/Library/Python/3.6/bin`
+#### PySide2
 
-</TabItem>
-</Tabs>
-
-
-### Installing PySide2 into python 3.6 for correct gui work
-
-AYON is using its own window widget inside Resolve, for that reason PySide2 has to be installed into the python 3.6 (as explained above).
-
-<Tabs
-  groupId="platforms"
-  defaultValue="win"
-  values={[
-    {label: 'Windows', value: 'win'},
-    {label: 'Linux', value: 'linux'},
-    {label: 'Mac', value: 'mac'},
-  ]}>
-
-<TabItem value="win">
-
-paste to any terminal of your choice
+AYON is using its own window widget inside Resolve, for that reason PySide2 has to be installed into the python.
 
 ```bash
-%LOCALAPPDATA%\Programs\Python\Python36\python.exe -m pip install PySide2
+cd <a path you had set to RESOLVE_PYTHON3_HOME>
+python.exe -m pip install PySide2
 ```
 
-</TabItem>
-<TabItem value="linux">
+### OpenTimelineIO
 
-paste to any terminal of your choice
+AYON is using OpenTimelineIO for editorial publishing. OpenTimelineIO has to be installed into the python.
 
 ```bash
-/opt/Python/3.6/bin/python -m pip install PySide2
-```
-
-</TabItem>
-<TabItem value="mac">
-
-paste to any terminal of your choice
 
 ```bash
-~/Library/Python/3.6/bin/python -m pip install PySide2
+cd <a path you had set to RESOLVE_PYTHON3_HOME>
+python.exe -m pip install OpenTimelineIO
 ```
 
-</TabItem>
-</Tabs>
-
-<div class="row markdown">
+### Debugging if Python is not working in Resolve
 
-### Set Resolve's Fusion settings for Python 3.6 interpereter
 
-<div class="col col--6 markdown">
+#### Set Resolve's Fusion settings for Python 3.6 interpreter
 
+In case Resolve is not showing Ayon menu, please have a look into Resolve's console and click to P3 tab. If it is not working then Python is not correctly connected to Resolve. One of the reasons might be that Fusion is not set to use Python 3.6 interpreter.
 
 As it is shown in below picture you have to go to Fusion Tab and then in Fusion menu find Fusion Settings. Go to Fusion/Script and find Default Python Version and switch to Python 3.6
```

Comment:
This could be worked around by the code in the OpenPype repository - you can force Resolve/Fusion to use the Py3 interpreter for the scripts no matter what if the file is named `.py3` instead of `.py` which might make it even safer pipeline-wise? Even though by now I'd definitely recommend Py3 anyway.